Makefile

```diff
@@ -11,8 +11,13 @@ clean: ## delete generated byte code and coverage reports
 	rm -rf assets
 	rm -rf pii_report
 
+COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
+.PHONY: $(COMMON_CONSTRAINTS_TXT)
+$(COMMON_CONSTRAINTS_TXT):
+	wget -O "$(@)" https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt || touch "$(@)"
+
```

Comment:
Hi, @stvstnfrd I had a question regarding this,
what was the problem that we faced with the previous implementation of using the `common_constraints` file directly?
or what problem does this (downloading the constraint file to local) solve?

Reply:
Hi @mraarif 

dependabot (the Github security upgrade PR creator, used across our repos) expects all requirements to be saved locally on disk and fails to fetch requirements files if they're remote.

So using `-c https://...` breaks the integration and dependabot stops working.

a little more text originally shared here:
- https://github.com/edx/xblock-lti-consumer/pull/168#issue-668597611